### DISCUSSION TITLE
Feature request: add description block for constraint

### DISCUSSION THREAD

**Discussion by yuchn** [flipt-io/flipt#1464] [Ideas]
As some of the constraints values are hard to remember, it would be great if we can have a description box in the 'Edit Constraint' panel. Or we may to have a generic mechanism for Flipt entities that need a description.
![image](https://user-images.githubusercontent.com/10615234/230274772-fb8fd06b-cd55-4b5c-bcbe-39d7b124823e.png)

**Comment by markphelps** [comment]
Great idea @yuchn !! I think we can start working on this ASAP

**Comment by markphelps** [comment]
@yuchn this is merged and will be out in the next release 🎉

**Reply by yuchn** [reply]
That is awesome! Thanks @markphelps! Looking forward to having a try.
